benchmarks/GraphBLAS-sharp.Benchmarks/Helpers.fs

```diff
@@ -16,35 +16,53 @@ type CommonConfig() =
 
     do
         base.AddColumn(
-            MatrixShapeColumn("RowCount", fun mtxReader -> mtxReader.ReadMatrixShape().RowCount) :> IColumn,
-            MatrixShapeColumn("ColumnCount", fun mtxReader -> mtxReader.ReadMatrixShape().ColumnCount) :> IColumn,
-            MatrixShapeColumn("NNZ", fun mtxReader -> mtxReader.ReadMatrixShape().Nnz) :> IColumn,
+            MatrixShapeColumn("RowCount", (fun mtxReader -> mtxReader.ReadMatrixShape().RowCount)) :> IColumn,
+            MatrixShapeColumn("ColumnCount", (fun mtxReader -> mtxReader.ReadMatrixShape().ColumnCount)) :> IColumn,
+            MatrixShapeColumn("NNZ", (fun mtxReader -> mtxReader.ReadMatrixShape().Nnz)) :> IColumn,
             TEPSColumn() :> IColumn,
             StatisticColumn.Min,
             StatisticColumn.Max
-        ) |> ignore
+        )
+        |> ignore
 
         base.AddJob(
-            Job.Dry
+            Job
+                .Dry
                 .WithWarmupCount(3)
                 .WithIterationCount(10)
                 .WithInvocationCount(3)
-        ) |> ignore
+        )
+        |> ignore
 
-type ClContext = ClContext of Brahma.FSharp.OpenCL.ClContext
-with
+type ClContext =
+    | ClContext of Brahma.FSharp.OpenCL.ClContext
```

Comment:
I hope, this code can be simplified using [these functions](https://github.com/YaccConstructor/Brahma.FSharp/blob/master/src/Brahma.FSharp.OpenCL.Core/ClContext.fs#L38).